contrib/nydusify/pkg/committer/commiter.go

```diff
@@ -84,7 +85,7 @@ func (cm *Committer) Commit(ctx context.Context, opt Opt) error {
 		return errors.Wrap(err, "parse target image name")
 	}
 
-	inspect, err := cm.manager.Inspect(ctx, opt.ContainerID)
+	inspect, err := cm.manager.Inspect(ctx, opt.ContainerID, opt.Namespace)
```

Comment:
How about calling `ctx = namespaces.WithNamespace(ctx, namespace)` before here.

Reply:
How about calling in `nydusify.go` `main` func?
Just in `return cm.Commit(c.Context, opt)`

Reply:
Better to put it in `committer.go`, so we can import it as a package in other  project. :)